lib/rules/define-props-declaration.js

```diff
@@ -14,10 +176,19 @@ module.exports = {
       categories: undefined,
       url: 'https://eslint.vuejs.org/rules/define-props-declaration.html'
     },
-    fixable: null,
+    fixable: 'code',
     schema: [
       {
         enum: ['type-based', 'runtime']
+      },
+      {
+        type: 'object',
+        properties: {
+          separateInterface: {
```

Comment:
Please rename to `autoFixToSeparateInterface` or similar.

Reply:
Also, please describe this option in the rule docs.

Reply:
done